Documentation/k8s-object.md

```diff
@@ -54,9 +54,26 @@ rook-ceph-rgw   10.3.0.177   <none>        53390/TCP   2m
 ```
 
 Now create the external service:
-```bash
-cd demo/kubernetes
-kubectl create -f rgw-external.yaml
```

Comment:
we're not being consistent about providing an instruction about how to use the inlined yaml.  in some places we say to save it as a specific file name then to `kubectl create` on that file and other places we just have the inline yaml with no specific instruction for what to do with the yaml content.